Example/DApp/Modules/Auth/AuthPresenter.swift

```diff
@@ -50,6 +51,33 @@ final class AuthPresenter: ObservableObject {
             }
         }
     }
+    
+    @MainActor
+    func deletePairing() {
+        Task {
```

Comment:
You do not need to create Task here, as you use @MainActor. Context is already async